dev/tests/integration/testsuite/Magento/SalesRule/Controller/Adminhtml/Promo/Quote/NewActionHtmlTest.php

```diff
@@ -31,6 +31,11 @@ class NewActionHtmlTest extends AbstractBackendController
      */
     private $formName = 'test_form';
 
+    /**
+     * @var string
+     */
+    private $jsFromObject = 'form';
```

Comment:
@Nazar65: there's a typo, should be `jsFormObject`, not `jsFromObject` 😉

Reply:
Nice catch :+1:  thank you